### ISSUE TITLE
New Feature Request: In-Project Stage Control Commands

### ISSUE THREAD

**Issue by SparkScratch-P** [SheepTester/htmlifier#112]
**New Feature Request:** In-Project Stage Control Commands
 Hi!
I have been seeing that u are active with this project, and it's pretty nice trying out the new features! I request one more feature for better usage.

  **Feature:** The best feature would be, if anyway... by blocks, or cloud vars, can press Start/Stop buttons from the project. whenever the project wants, it should be able to start and stop the project or reboot it without reloading.

 Hope it gets added soon.

**Thank You**

**Comment by SheepTester** [comment]
You can already stop the project using the stop all project. For compatibility with Scratch, you probably should use broadcasts to initialise your project instead of the `when green flag clicked` block, then when the green flag is clicked, and whenever you want to start the project again, send the broadcast. This allows for maximal customisability, and it also works in Scratch

**Comment by SparkScratch-P** [comment]
Does the stop all block also stops the work of the 'Ask and wait' block
also? Then, that is the perfect aid.
[image: GeoTrack]
<https://geotrack.email/?utm_source=gmail&utm_medium=signature&utm_campaign=viralSignature>
Sender
notified with GeoTrack
<https://geotrack.email/?utm_source=gmail&utm_medium=signature&utm_campaign=viralSignature>
[image: 8]

**Comment by SheepTester** [comment]
It should, I believe.

**Comment by SparkScratch-P** [comment]
Yes, it does. Thanks for the reminder,😄 after getting detached from my
scratch account, i am forgetting many small tricks. 😕
[image: GeoTrack]
<https://geotrack.email/?utm_source=gmail&utm_medium=signature&utm_campaign=viralSignature>
Sender
notified with GeoTrack
<https://geotrack.email/?utm_source=gmail&utm_medium=signature&utm_campaign=viralSignature>
